content/doc/book/security/securing-jenkins.adoc

```diff
@@ -12,30 +13,7 @@ ifndef::env-github[:imagesdir: ../../resources]
 :toc: left
 endif::[]
 
-= Securing Jenkins
-
-Securing Jenkins has two aspects to it.
-
-* Access control, which ensures users are authenticated when accessing Jenkins
-  and their activities are authorized.
-* Protecting Jenkins against external threats
-
-== Access Control
-
-You should lock down the access to Jenkins UI so that users are authenticated
-and appropriate set of permissions are given to them. This setting is
-controlled mainly by two axes:
-
-* *Security Realm*, which determines users and their passwords, as well as what
-  groups the users belong to.
-* *Authorization Strategy*, which determines who has access to what.
-
-These two axes are orthogonal, and need to be individually configured. For
-example, you might choose to use external LDAP or Active Directory as the
-security realm, and you might choose "everyone full access once logged in" mode
-for authorization strategy. Or you might choose to let Jenkins run its own user
-database, and perform access control based on the permission/user matrix.
-
+This page is a placeholder for links to some old Wiki pages until we can properly dispose of this information.
 
 * https://wiki.jenkins.io/display/JENKINS/Quick+and+Simple+Security[Quick and Simple Security] --- if you are running Jenkins like `java -jar jenkins.war` and only need a very simple setup
 * https://wiki.jenkins.io/display/JENKINS/Standard+Security+Setup[Standard Security Setup] --- discusses the most common setup of letting Jenkins run its own user database and do finer-grained access control
```

Comment:
> Immediately after installation, Jenkins will allow anyone to run anything as user jenkins, which is bad.

Yeah, hasn't been true for more than 5 years. Some basic configuration of various security realms (Jenkins User Database, Active Directory Plugin, PAM Auth Plugin) and authorization strategies (Matrix). We may be able to grab some of the detail content but mostly for plugin docs. No need to keep a visible link around, a TODO comment will do.

Reply:
Cannot do line comments on the rest, so adding here:

### Apache frontend for security

Discusses how to compile Apache 2.2 with non-standard options (superseded since 2012, obsolete since 2017), then how to set up a reverse proxy with security (similar to above), which we probably don't think is great.

Lastly, running Jenkins with AJP, which hasn't been possible since 2.0 in 2016. Delete the link.

### Authenticating scripted clients

Migrated to https://www.jenkins.io/doc/book/system-administration/authenticating-scripted-clients/ . It's just a "how to use Jenkins" think that happens to touch on a security topic. Delete the link.

### Matrix-based security

Very minimal page (a list of permissions and their descriptions). The only nontrivial content is about the "dangerous" permissions that are obsolete since 2.222.x (March 2020). Delete the link.

Reply:
So this entire page can be deleted 🎉

Reply:
Hooray!  File is deleted and @MarkEWaite has been alerted to your comments about the status of these files.